blog.html

```diff
@@ -56,7 +56,7 @@ <h3>Blog</h3>
                 <div class="header-left-bottom">
                     <img src="./images/words.png" alt="#">
                 </div>
-                <div class="header-right-bottom">
+                <div class="header-right-bottom catagor">
```

Comment:
Spelling is not correct
```suggestion
                <div class="header-right-bottom category">
```